master/internal/webhooks/shipper_test.go

```diff
@@ -0,0 +1,184 @@
+//go:build integration
+
+package webhooks
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/atomic"
+	"golang.org/x/exp/maps"
+
+	"github.com/determined-ai/determined/master/internal/db"
+	"github.com/determined-ai/determined/master/pkg/model"
+	"github.com/determined-ai/determined/master/pkg/schemas"
+	"github.com/determined-ai/determined/master/pkg/schemas/expconf"
+)
+
+const (
+	pathToMigrations = "file://../../static/migrations"
```

Comment:
I believe we already have a constant for that: https://github.com/determined-ai/determined/blob/master/master/internal/cache/cache_intg_test.go#L21

Reply:
updated!

Reply:
i feel it is _technically_ correct for this to be a package specific constant, since it is the path to migration from the pacakge